### PR TITLE
View table

### PR DESCRIPTION
**Pull request by LindsayYoung**
[View Complainant's Name and Contact information on the View All Table](https://app.zenhub.com/workspaces/doj-crt-intake-scrum-board-5d03bf56c1c8a35d482eca0f/issues/18f/crt-portal/26)

## What does this change?
Adds an authenticated view all page
## Screenshots (for front-end PR):
View all table:
/form/view
Screen Shot 2019-09-17 at 1 43 43 PM

Login screen:
(it redirects here if not logged in)
Screen Shot 2019-09-16 at 2 36 21 PM

## Checklist:
From the UX 
- run locally
  - [ ] in an incognito window (or however you want to be not logged in) try to visit `/form/view` it shoudl redirect you to a login page
  - [ ] login with a user
  - [ ] look at the view all page and make sure it is follows the [expected UX](https://docs.google.com/presentation/d/1k0HWEhcIb6QC5fXNaXmePqoFpKau6_d-mZL2Q4DCa-w/edit#slide=id.g611277aa76_0_172) Keeping in mind we don't have all the data points yet and sorting, filtering, pagination, and lots of the fields are out of scope.  
+ [ ] tests pass https://circleci.com/gh/18F/crt-django/140

## Notes for reviewer:
You need some reports locally to have anything in the view table

### CODE REVIEW

**Comment by LindsayYoung** [comment]
- [ ] Text: Complainant name/contact > "contact" name/contact
- [ ] Text: Submitted (instead of submission date)
- [ ] Email and phone fixed to one line, no wrapping
- [ ] If blank field, "Blank" italicized so can tell if someone wrote nothing vs wrote nothing. Suggesting blank so that we can include a filter sort "include blank" type thing.
- [ ] Title change to - "CRT Complaint Records"
- [ ] Remember to check in IE 11!

File: Pipfile [hunk 1]
```diff
@@ -11,7 +11,7 @@ django = "==2.2.4"
 gunicorn = "*"
 django-compressor = "*"
 django-compressor-toolkit = "*"
-boto3 = "*"
+boto3 = "==1.9.215"
```
Comment (alexsoble) [inline Pipfile]: @LindsayYoung Is this change related to the View Table or unrelated?

File: crt_portal/templates/registration/login.html [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  <div class="grid-container">
+    <div class="grid-col-1"></div>
+        <h2>Login</h2>
+        <p>Please note this process will be replaced with DOJ single sign on.</p>
+        <form label="log in form" method="post">
+          {% csrf_token %}
+          {{ form.as_p }}
+          <button label="login" type="submit">Login</button>
+        </form>
+  <div class="grid-container">
```
Comment (alexsoble) [inline crt_portal/templates/registration/login.html]: Looks like these should be closing tags for the divs above.

Comment (LindsayYoung) [reply crt_portal/templates/registration/login.html]: yea, feel free to fix that

File: Pipfile (hunk 1 above)
Comment (LindsayYoung) [reply Pipfile]: I was having deploy issues when I was checking this, so I added that, I think it was probably the AWS region variable name change. We can git rid of this change if you want.

File: crt_portal/templates/registration/login.html (hunk 2 above)
Comment (alexsoble) [reply crt_portal/templates/registration/login.html]: Fixed!

File: Pipfile (hunk 1 above)
Comment (alexsoble) [reply Pipfile]: Hmmmm, since it's not related to the view table feature, I can back it out. We can introduce this change in another PR if it's important to getting the deploy process working.

Comment (alexsoble) [reply Pipfile]: Done

**Comment by alexsoble** [comment]
# Screenshots! 

Ran a few super quick manual a11y checks on the page.

Here's a "default" view:

default

## Zoomed in

Here's a zoomed-in view at 200%. The whole table fits beautifully!: 



## Grayscale 

Here's a grayscale view. Totally legible.

bnw

**Comment by alexsoble** [comment]
One remaining issue is the footer, which is not sticky and which therefore looks very weird on the login page: 

sticky-footer

I'll address this next week and will make sure the solution is Internet Explorer friendly!
